upup/pkg/fi/cloudup/gce/network.go

```diff
@@ -31,9 +31,16 @@ import (
 
 // UsesIPAliases checks if the cluster uses IP aliases for network connectivity
 func UsesIPAliases(c *kops.Cluster) bool {
+	// "GCP" networking mode is called "ip-alias" or "vpc-native" on GKE.
 	if c.Spec.Networking.GCP != nil {
 		return true
 	}
+
+	if c.Spec.Networking.Kindnet != nil {
+		// TODO: Are we _always_ using ipalias - should we at least check the cloud is GCP?
```

Comment:
yeah, I don't want kindnet to use overlays, there are already plenty of options ... besides, we'll focus on IPv6 for development that will not require overlays as there is plenty of space

Reply:
Great - I think for (pure) IPv6 we probably don't want to trigger the configuration of alias IP ranges on the instances here, but ... let's see how it goes.